### ISSUE TITLE
Doc: getting started for new user on Mac is not straightforward/ obvious

### ISSUE THREAD

**Issue by matcomer** [galasa-dev/projectmanagement#1541]
## Describe the bug
The Docs simply say: 
'After downloading the binary, re-name it to galasactl (for Mac or Unix) or galasactl.exe (for Windows) and put it on your PATH.'

## Steps to reproduce
I wish it was that easy!
1. Please provide some background guidance on how to access the PATH file and I would suggest an example (I didn't know my PATH file is provided in a file called paths in /private/etc)
2. For me, I edited the paths file to add a new directory containing my renamed galasactl file. I also had to chmod the galasctl file to give access to execute it
3. Then I was not able to run/ execute it via the CLI. Instead I would get an error message saying galasctl Cannot Be Opened Because the Developer Cannot be Verified'
4. After googling the problem, I discovered I needed to go 'Developer tools' in Privacy & Settings within System Settings, to change the security policy for this file. This must be documented
5. Even then I was not able to run the file directly via the CLI, I still would get the same error
6. The only way to get started, was to open the file via Finder and accept the warning which was successful
7. Once I had been successful once (by following the above steps), I am now able to run galasactl from my terminal

This feels rather clunky.
In short term some doc guidance is necessary to help someone else through this step.
Longer term, is there away to avoid this issue altogether by enabling the file to be trusted by Apple?

**Comment by CaroMac** [comment]
Thank you for the comments. PR https://github.com/galasa-dev/galasa.dev/pull/689/files is created to address issues.
